spec/popup/format-spec.js

```diff
@@ -0,0 +1,51 @@
+import format from '../../src/common/popup/utils/format.js';
+
+describe('format util', () => {
+  const ticket = {
+    id: 'BTC-042',
+    title: 'Weird Bug while eating cake',
```

Comment:
Maybe use a more realistic title? E.g. `'Add more tests for src/common/popup/utils/format.js'`